src/qt/pivx/forms/masternodeswidget.ui

```diff
@@ -226,7 +226,7 @@
                  <string notr="true"/>
                 </property>
                 <property name="text">
-                 <string>No active Master Node yet</string>
+                 <string>No active Masternode yet</string>
```

Comment:
This is actually a dummy string (it is set in the `.cpp` file to it's visible value).

Rather than have a duplicate string here, would change this line to read
```html
<string notr="true">N/A</string>
```